cyberdrop_dl/managers/client_manager.py

```diff
@@ -475,15 +473,20 @@ def check_file_duration(self, media_item: MediaItem) -> bool:
         if not (is_video or is_audio):
             return True
 
-        def get_duration() -> float | None:
+        async def get_duration() -> float | None:
             if media_item.duration:
                 return media_item.duration
-            props: dict = {}
+
+            probe_path = media_item.complete_file if media_item.downloaded else media_item.url
+            probe_headers = self.download_client._get_download_headers(media_item.domain, media_item.referer)
+            properties: FFprobeResult = await probe(probe_path, headers=probe_headers)
```

Comment:
Only parse headers for URLs
```suggestion
            if media_item.downloaded:
                probe_path = media_item.complete_file
                probe_headers = None 
            else:
                probe_path = media_item.url
                probe_headers = self.download_client._get_download_headers(media_item.domain, media_item.referer)
            properties: FFprobeResult = await probe(probe_path, headers=probe_headers)
```